appveyor.yml

```diff
@@ -1,6 +1,14 @@
+os: Visual Studio 2015 CTP
+
 init: 
 build_script: 
   - cmd: appveyor-build.cmd
+
+# scripts that run after cloning repository
+install:
+  # by default, all script lines are interpreted as batch
+  - set PATH=C:\Program Files (x86)\MSBuild\14.0\Bin;%PATH%
```

Comment:
`C:\` drive should not be hardcoded.
`Program Files (x86)` should not be hardcoded.